UI/static/main.css

```diff
@@ -0,0 +1,82 @@
+body{
+    margin: 0px;
```

Comment:
I would advise you make you set `margin:auto` for better body balance horizontally